sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -89,7 +89,7 @@ def begin_recognize_receipts(self, stream, **kwargs):
         if content_type is None:
             content_type = get_content_type(stream)
 
-        return self._client.begin_analyze_receipt_async(
+        return self._client.begin_analyze_receipt_async(  # type: ignore
```

Comment:
@iscai-msft is there anyway to change the typing for the `begin_` generated code to have the return type as `LROPoller` instead of None? I'm needing to ignore the types on these methods

Reply:
you should import LROPoller (in a if type_checking block if sync), and then you can use it like # type: LROPoller https://github.com/Azure/azure-sdk-for-python/blob/8b81cc6fd645b68f8317638ffca23563ac6c602c/sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py#L60

Reply:
I'm talking about in the generated code, though. I do have the LROPoller type hint. The problem is I say that it returns LROPoller, but the generated code says `None`

Reply:
Oh I'm sorry I totally misread your comment. I actually have a PR that should be going in today (fingers crossed) that will fix that: https://github.com/Azure/autorest.python/pull/507/files#diff-5da12f8baec7520bf65f3def1b84b390R105